### ISSUE TITLE
Math.Sqrt and unsigned integers don't work.

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#2399]
### Expected

```
3
```

### Actual

The call is ambiguous between the following methods or properties: `Math.Sqrt(double)` and `Math.Sqrt(decimal)`

### Steps To Reproduce

http://deck.net/0d560f85537c64e5fa0f0c2a527dc045
https://dotnetfiddle.net/z8PUpW

```c#
using System;
					
public class Program
{
	public static void Main()
	{
		Console.WriteLine(Math.Sqrt(9u));
	}
}

**Comment by geoffreymcgill** [comment]
Hi @michaelcheers,

Thanks for bringing this defect to our attention. We have fixed the issue and submitted pull request #2402 with the changes. Once reviewed and tested, the pull request will be merged into the **master** branch.

**Comment by LeonidVeriga** [comment]
The fix is:
-  Math.Sqrt(decimal d) is removed because it is not presented in .NET API
-  Math.Sqrt(int d) is removed because it is not presented in .NET API
